app/models/like.rb

```diff
@@ -0,0 +1,7 @@
+class Like < ApplicationRecord
+  belong_to :posts, :users
+
+  def update_comment_counter
+    posts.likes_counter += 1
+  end
+end
```

Comment:
- [ ] Likes belong to the author as well, please implement it.
- [ ] here you can use class_name: 'User' instead of :users